### PR TITLE
Social Links Working and designing

### LINKED ISSUE #37: [Issue] Social Links in Footer Not Working
## Description:
The Social Links in the footer section are not working. The images do not redirect to any social website. These links should take users to the correct social media pages, but nothing happens when clicked. I will fix this issue so that all links work properly.

Screenshot (354)
## Steps to Reproduce:

1. Navigate to the footer section of the website.

2. Click on social media icons (Facebook, Twitter, Instagram, etc.).

3. Observe that the links do not navigate to the respective social media pages.

### PR DESCRIPTION
**Pull request by KhawajaFashi**
## Type of PR
- [X] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to solve the issue that social links have. When they were clicked they should be redirected to their respective social platforms which they did not. 

## Screenshots / Videos
**Before:**
- Social Links were only images they do not redirect to any social platform.

https://github.com/user-attachments/assets/c91a77e7-425e-44bc-86f5-ce7b4ca1252f

**After:**
- Social links redirect to their respective platforms and they have been beautified.

https://github.com/user-attachments/assets/ad1317f8-e20c-44f8-8e80-46e48e4f78b0

## Checklist
- [X] I have performed a self-review of my code.
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

Thank you for reviewing my pull request!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -269,10 +269,18 @@ <h4>Our Address</h4>
                 <div class="footer-section">
                     <h4 class="follow">Follow Us on</h4>
                     <div class="social-links">
-                        <img src="images/fb.png" alt="Facebook" />
-                        <img src="images/tw.png" alt="Twitter" />
-                        <img src="images/ig.png" alt="Instagram" />
-                        <img src="images/linkdin.png" alt="linkdin" />
+                        <a href="https://www.facebook.com/" target="_blank">
```
Comment (swaraj-das) [inline index.html:272]: If you add the icons from external links, please remove the previous four social media images from images folder.

File: index.html [hunk 2]
```diff
@@ -269,10 +269,18 @@ <h4>Our Address</h4>
                 <div class="footer-section">
                     <h4 class="follow">Follow Us on</h4>
```
Comment (swaraj-das) [inline index.html:270]: Please check if it's possible not to add the Instagram icon image and use the external link only.

File: index.html (hunk 1 above)
Comment (KhawajaFashi) [reply index.html:272]: ok sir

File: index.html (hunk 2 above)
Comment (KhawajaFashi) [reply index.html:270]: Actually instagram logo has gradient color so it is difficult to set its color.

**Comment by swaraj-das** [comment]
Merge Done

**Comment by KhawajaFashi** [comment]
@swaraj-das sir can you please fix the label (level 1) it should be without spaces

**Pull request merged**